platform/ui/src/utils/clickOutsideHandler.js

```diff
@@ -1,7 +1,8 @@
-export default element => {
+export default (element, callback = () => {}) => {
   const handleClickOutside = event => {
     if (element.current && !element.current.contains(event.target)) {
       element.current.blur();
+      callback();
```

Comment:
`onClickOutside` and assume it's set?
Or create a separate `blueHandlerListener` if we're trying to use this for both things?

Reply:
They were split into two separate methods 👍  Even though they seem to do the same thing, one is related to blur, and the other to click outside an element. Thanks!